src/NextGen/dataHandler/dataHandler.ts

```diff
@@ -0,0 +1,7 @@
+import { IMParticleComponent } from '../core/component';
+
+export interface IMParticleDataHandler extends IMParticleComponent {
```

Comment:
@markvdouw - can you clarify the difference between data handler and data handler strategy from a 30k foot view?  I'm having a hard time understanding the distinction.

Reply:
The DataHandler defines the operation that must be implemented by the class that hold the concrete strategies.
The DataHandlerStrategy defines the operations that must be implemented by each concrete strategy.

Example:
DataManager (strategies : List<DataHandlerStrategy>) implements DataHandler 

In this example makes sense that the DataHandlerStrategy defines all the operations in DataHandler PLUS any other specific operation needed in the concrete strategies (for example to map/convert data)
https://github.com/mParticle/mparticle-android-sdk/pull/367/files#diff-1e033fd6678f20a7e1dd496e9e3d1818f5241f811511b3a4891e3c811104b75a